Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.13'
+
+pipeline {
+  agent { label 'linux' }
+
+  options {
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  stages {
+    stage('Build') {
+      steps {
+        script {
+          nix.flake("default")
```

Comment:
I would add a second stage where you run the built binary with `--version` like we do in `go-waku`:
https://github.com/status-im/go-waku/blob/4ef460cb951a72b2a0be2b3f23347ef8f569ea08/ci/Jenkinsfile.nix-flake#L33-L37